drop-in/src/main/java/com/adyen/checkout/dropin/ComponentParsingProvider.kt

```diff
@@ -186,6 +192,7 @@ internal fun getProviderForType(type: String): PaymentComponentProvider<PaymentC
         PaymentMethodTypes.SCHEME -> CardComponent.PROVIDER
         PaymentMethodTypes.SEPA -> SepaComponent.PROVIDER
         PaymentMethodTypes.WECHAT_PAY_SDK -> WeChatPayComponent.PROVIDER
+        PaymentMethodTypes.BLIK -> BlikComponent.PROVIDER
```

Comment:
A bit nit picky, but maybe add this in alphabetic order, the same order the modules show up on the IDE, to make this ugly code a bit easier to read at least 😅

Reply:
Thanks, I didn't notice they were sorted 😅